pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -416,14 +422,15 @@ func (p *PriorityQueue) Update(oldPod, newPod *v1.Pod) error {
 		oldPodInfo := newQueuedPodInfoNoTimestamp(oldPod)
 		// If the pod is already in the active queue, just update it there.
 		if oldPodInfo, exists, _ := p.activeQ.Get(oldPodInfo); exists {
-			p.nominatedPods.update(oldPod, newPod)
+			// TODO: make update() public?
+			p.PodNominator.(*NominatedPodMap).update(oldPod, newPod)
```

Comment:
+1 for make update() public.

Reply:
lets make it public and name it ```UpdateNominatedPod``` for now.

Reply:
I was hesitant on this is because the plugin developer would only need the current two methods:

```go
type PodNominator interface {
	UpdateNominatedPodForNode(pod *v1.Pod, nodeName string)
	DeleteNominatedPodIfExists(pod *v1.Pod)
}
```

If we made this update() method public, it has to be added to PodPreemptor interface, so users would be confused on which UpdateXYZ() method to use. WDYT?

Reply:
nvm, I renamed `UpdateNominatedPodForNode` to `AddNominatedPod`, so introducing `UpdateNominatedPod` won't be ambigurous.